CONTRIBUTING.md

```diff
@@ -6,6 +6,8 @@
 * Use the same coding style as the rest of the
 [codebase](https://github.com/bookshelf/bookshelf/blob/master/src/bookshelf.js).
 
+* Use the [issue](https://github.com/bookshelf/bookshelf/blob/master/ISSUE-TEMPLATE.js) or [pull request](https://github.com/bookshelf/bookshelf/blob/master/PR-TEMPLATE.js) templates when creating one entry, this will make clear to everyone the scope of your proposition or the subject of the issue
```

Comment:
Missing period at the end of the sentence. Also, the English used is a bit weird. I would change the following:

> ...templates when creating **your** entry, this will **help clarify the** scope of your **proposal** or the subject of **your** issue.